### PR TITLE
review: harden checks correctness output

### PR DESCRIPTION
**Pull request by WUAIBING**
## Summary
- harden bridge publishability checks for Checks performed entries that cite unsupported identifiers
- replace no-finding and approval check summaries with deterministic grounded defaults from task context
- drop partial trailing word clips that degrade published review quality

## Validation
- python -m pytest tests/test_github_bridge.py tests/test_node_runtime.py

### CODE REVIEW

**Comment by WUAIBING** [comment]
@Hub-Sentinel review this PR

File: node/mep_runtime.py [hunk 1]
```diff
@@ -668,6 +668,19 @@ def _clean_review_list(values: Any, *, max_items: int, max_chars: int) -> list[s
     return cleaned
 
 
+def _clip_without_partial_token(text: str, *, max_chars: int) -> str:
```
Comment (Hub-Sentinel) [inline node/mep_runtime.py:671]: **`_clip_without_partial_token` returns empty string when the first word exceeds `max_chars`, potentially dropping meaningful review text.**

The function uses `text.rfind(' ', 0, max_chars)` to find a word break. If no space exists in the first `max_chars` characters (e.g., a single long word), `rfind` returns -1, and the function returns `text[:max_chars]` which is empty. This could silently remove important content from reviews.

**Review by Hub-Sentinel — COMMENTED** [review]
## Review Findings

The PR hardens bridge publishability checks by adding `_list_entries_with_unsupported_identifiers` to filter out check entries referencing unsupported identifiers, and introduces `_clip_without_partial_token` to avoid partial-word clipping in review text. Tests are updated to cover these changes. No high-impact correctness or regression issues were found in the diff.

Observation: In `node/mep_runtime.py`, `_clip_without_partial_token` uses `text.rfind(' ', 0, max_chars)` to find a word break; if the first word exceeds `max_chars`, `rfind` returns -1, and the function returns an empty string. This edge case is not handled, potentially dropping meaningful text for long single-word inputs.

Touched paths reviewed: `bridge/github_to_mep.py`, `node/mep_runtime.py`, `tests/test_github_bridge.py`, `tests/test_node_runtime.py`

Tests reviewed: `tests/test_github_bridge.py`, `tests/test_node_runtime.py`

Risk areas checked: Correctness of new filtering logic for unsupported identifiers, Edge-case handling in text clipping function, Test coverage for new methods and fallback behavior

Checks performed: Verified that `_list_entries_with_unsupported_identifiers` uses `patch_info.get('changed_identifiers', [])` to safely ha, Checked that `_clip_without_partial_token` returns empty string when no space found before `max_chars`, Reviewed test assertions for `test_status_callback_salvages_checks_with_unsupported_identifier_by_stripping_checks_secti, Examined test `test_structured_review_renders_risk_coverage_for_no_finding_reviews` for realistic identifier usage

Changed identifiers verified: `_list_entries_with_unsupported_identifiers`, `_clip_without_partial_token`, `unsupported_check_entries`, `word_break`

1. **`_clip_without_partial_token` returns empty string when the first word exceeds `max_chars`, potentially dropping meaningful review text.** (`node/mep_runtime.py`): The function uses `text.rfind(' ', 0, max_chars)` to find a word break. If no space exists in the first `max_chars` characters (e.g., a single long word), `rfind` returns -1, and the function returns `text[:max_chars]` which is empty. This could silently remove important content from reviews.

**Comment by WUAIBING** [comment]
Superseded by #313, which consolidates this slice with the rebased reviewer-correctness stack on top of merged main and keeps the combined bridge/runtime regressions green.
